proto/pocket/migration/query.proto

```diff
@@ -58,3 +71,20 @@ message QueryAllMorseClaimableAccountResponse {
            cosmos.base.query.v1beta1.PageResponse pagination            = 2;
 }
 
+// QueryMorseClaimableAccountRequest is used to query for MorseClaimableAccounts by Morse address.
+// If the shannon_address (or claimed_at_height) for any given MorseClaimableAccount
+// is empty, it has not been claimed.
+// See the comments for MorseClaimableAccount for more context.
+message QueryMorseClaimableAccountsByShannonAddressRequest {
+  // The bech32-encoded address of the Shannon account to which one or more Morse accounts were claimed.
+  string shannon_address = 1;
+
+  cosmos.base.query.v1beta1.PageRequest pagination = 2;
```

Comment:
Can we avoid needing pagination for this? I do expect the list to be that big per address.